src/Microsoft.ApplicationInsights/NuGet/Package.nuspec

```diff
@@ -40,32 +40,32 @@
   </metadata>
   <files>
     <!-- Binaries for .NET Standard 1.3 -->
-    <file src="$configuration$\Core\Managed\netstandard1.3\Microsoft.ApplicationInsights.dll" target="lib\netstandard1.3" />
-    <file src="$configuration$\Core\Managed\netstandard1.3\Microsoft.ApplicationInsights.pdb" target="lib\netstandard1.3" />
-    <file src="$configuration$\Core\Managed\netstandard1.3\Microsoft.ApplicationInsights.xml" target="lib\netstandard1.3" />
+    <file src="$configuration$\Microsoft.ApplicationInsights\Managed\netstandard1.3\Microsoft.ApplicationInsights.dll" target="lib\netstandard1.3" />
```

Comment:
I remember we had an issue with the long paths in the nuget packages location (reference file paths were too lengthy within the application being on-boarded in VS), and here we add ~25 extra characters. However, if this nuget will mostly be installed with Microsoft.ApplicationInsights.Web - then we should be OK.